### ISSUE TITLE
Error: No such file or directory - unzip

### ISSUE THREAD

**Issue by mmistakes** [benbalter/jekyll-remote-theme#5]
Guessing this is likely a Windows issue, but wanted to log it just in case I'm doing something wrong.

On a new `jekyll new` site:

1. Add `gem "jekyll-remote-theme"` to `Gemfile`
2. Remove `theme: minima` and replace with `remote_theme: mmistakes/jekyll-theme-basically-basic`
3. Add `jekyll-remote-theme` to `plugins` array in _config.yml
4. Run `bundle exec jekyll build --verbose`

```
E:\Users\Michael.Rose\Documents\Repositories\remote-theme-test>bundle exec jekyll build --verbose
  Logging at level: debug
Configuration file: E:/Users/Michael.Rose/Documents/Repositories/remote-theme-test/_config.yml
         Requiring: jekyll-feed
         Requiring: jekyll-remote-theme
         Requiring: kramdown
            Source: E:/Users/Michael.Rose/Documents/Repositories/remote-theme-test
       Destination: E:/Users/Michael.Rose/Documents/Repositories/remote-theme-test/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
      Remote Theme: Using theme mmistakes/jekyll-theme-basically-basic
      Remote Theme: Downloading https://codeload.github.com/mmistakes/jekyll-theme-basically-basic/zip/master to E:/Users/MICHAE~1.ROS/AppData/Local/Temp/jekyll-remote-theme-20171020-27928-1x22e5m.zip
      Remote Theme: Unzipping E:/Users/MICHAE~1.ROS/AppData/Local/Temp/jekyll-remote-theme-20171020-27928-1x22e5m.zip to E:/Users/MICHAE~1.ROS/AppData/Local/Temp/jekyll-remote-theme-20171020-27928-4jguu7
jekyll 3.6.1 | Error:  No such file or directory - unzip
```

Will report back the experience on mac OS when I get a chance to test it.

**Comment by pathawks** [comment]
Probably related to #3

Does Windows not have any `unzip` command at all?

**Comment by benbalter** [comment]
@mmistakes if you are able to change your project's Gemfile to the following and run `bundle update`, it *might* work:

```ruby
gem 'jekyll-remote-theme', github: 'benbalter/jekyll-remote-theme', branch: 'rubyzip'
```

**Comment by mmistakes** [comment]
:+1: That does the trick. No more errors on Windows and the remote theme files are pulled in as expected.

**Comment by benbalter** [comment]
@mmistakes 🤘 Thanks for testing it out. Released [v0.2.0](https://github.com/benbalter/jekyll-remote-theme/releases/tag/v0.2.0) which should include both #7 as well as #6 which should help Windows compatability (pure Ruby versus shelling out to command line).
